plugins/apollo-explorer/src/components/EmbeddedExplorer/EmbeddedExplorer.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+// import { Typography, Grid } from '@material-ui/core';
+import {
+  // InfoCard,
+  Header,
+  Page,
+  Content,
+  // ContentHeader,
+  HeaderLabel,
+  // SupportButton,
+} from '@backstage/core-components';
+import { ApolloExplorerReact } from '@apollo/explorer';
+import './style.css';
+
+export const EmbeddedExplorer = () => (
+  <Page themeId="tool">
+    <Header title="Apollo Studio Explorer">
+      {' '}
+      {/* subtitle="Optional subtitle"> */}
+      <HeaderLabel label="Owner" value="Apollo GraphQL" />
+      <HeaderLabel label="Lifecycle" value="Alpha" />
+    </Header>
+    <Content noPadding stretch>
+      <ApolloExplorerReact
+        className="embeddableExplorer"
+        graphRef="acephei@current"
+        endpointUrl="https://acephei-gateway.herokuapp.com"
```

Comment:
This doesn't seem to be accessible, could we use a publicly accessible example API instead?

Did you have a look at how the GraphiQL plugin sets up the endpoints as well btw? Feeling that it's quite important to have configurable endpoints with authentication for internal use-cases.